src/Controls/src/Core/Platform/Windows/VisualElementTracker.cs

```diff
@@ -156,9 +156,17 @@ void HandleDragOver(object sender, Microsoft.UI.Xaml.DragEventArgs e)
 					return;
 				}
 
+				Point position = Point.Zero;
+
+				if (e.OriginalSource is UIElement element)
+				{
+					var result = e.GetPosition(element);
+					position = new Point(result.X, result.Y);
```

Comment:
Actually, we do the if check a lot too, you could fix this with an extension method on the OriginalSource as follows:
```csharp
public static Point GetPosition (this Microsoft.UI.Xaml.DragEventArgs self) {
	if (e.OriginalSource is UIElement element)
	{
		var result = e.GetPosition(element);
		return new Point(result.X, result.Y);
	}
	return Point.Zero;
}
```

Then replace all the same code for:
```
var position = e.GetPosition();
```
We have 5 occurrences.